src/plugins/ini/ini.c

```diff
@@ -23,17 +26,20 @@ int elektraIniClose(Plugin *handle, Key *parentKey);
 
 #include "contract.h"
 
+#define INTERNAL_ROOT_SECTION "GLOBALROOT"
```

Comment:
only if no section key `user/inimountpoint/section1`  - ( `[section1]` ) exists, otherwise kdb set would just overwrite the key, and i'm not sure if it would be possible to restore it.